### ISSUE TITLE
Uncaught ChunkedEncodingError during importcontent

### ISSUE THREAD

**Issue by ivanistheone** [learningequality/kolibri#4496]
### Observed behavior

importchannel/importcontent management command fails 

### User-facing consequences

Users would have their import process interrupted and have to resume.

### Errors and logs

```
[104.196.110.174] sudo: python /kolibrihome/kolibri-0.11.0b2.pex manage importchannel network cfa63fd45abf5b7390b1a41f3b4971bb
[104.196.110.174] sudo: python /kolibrihome/kolibri-0.11.0b2.pex manage importcontent network cfa63fd45abf5b7390b1a41f3b4971bb

Fatal error: sudo() received nonzero return code 1 while executing!

Requested: python /kolibrihome/kolibri-0.11.0b2.pex manage importcontent network cfa63fd45abf5b7390b1a41f3b4971bb
Executed: sudo -S -p 'sudo password:'  -u "kolibri"  /bin/bash -l -c "export KOLIBRI_HOME=\"/kolibrihome\" && python /kolibrihome/kolibri-0.11.0b2.pex manage importcontent network cfa63fd45abf5b7390b1a41f3b4971bb"

================================================================== Standard output ==================================================================

INFO     Running Kolibri with the following settings: kolibri.deployment.default.settings.base
ERROR    An error occurred during content import: ('Connection broken: error("(104, \'ECONNRESET\')",)', error("(104, 'ECONNRESET')",))1945371.66it/s]
INFO     Setting availability of 257 LocalFile objects based on passed in checksums                   | 2097152/3414226 [00:00<00:00, 15455603.43it/s]
INFO     Setting availability of File objects based on LocalFile availability
INFO     Setting availability of non-topic ContentNode objects based on File availability
INFO     Setting availability of ContentNode objects with children for 4 levels
INFO     Setting availability of ContentNode objects with children for level 4
INFO     Setting availability of ContentNode objects with children for level 3
INFO     Setting availability of ContentNode objects with children for level 2
INFO     Setting availability of ContentNode objects with children for level 1
INFO     Setting totals of coach content ContentNode objects with children for 4 levels
INFO     Setting totals of coach content ContentNode objects with children for level 4
INFO     Setting totals of coach content ContentNode objects with children for level 3
INFO     Setting totals of coach content ContentNode objects with children for level 2
INFO     Setting totals of coach content ContentNode objects with children for level 1
 68%|███████████████████████████████████████████████████████████████▏                             | 802880109/1181502815 [01:47<00:50, 7466685.63it/s]
Traceback (most recent call last):
  File ".bootstrap/_pex/pex.py", line 365, in execute
  File ".bootstrap/_pex/pex.py", line 293, in _wrap_coverage
  File ".bootstrap/_pex/pex.py", line 325, in _wrap_profiling
  File ".bootstrap/_pex/pex.py", line 408, in _execute
  File ".bootstrap/_pex/pex.py", line 466, in execute_entry
  File ".bootstrap/_pex/pex.py", line 471, in execute_module
  File "/usr/lib/python2.7/runpy.py", line 180, in run_module
    fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/__main__.py", line 10, in <module>
    main(args=sys.argv[1:])
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/utils/cli.py", line 642, in main
    manage(command, args=django_args)
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/utils/cli.py", line 432, in manage
    execute_from_command_line(argv=argv)
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/dist/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/dist/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/dist/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/dist/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/core/tasks/management/commands/base.py", line 99, in handle
    return self.handle_async(*args, **options)
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/core/content/management/commands/importcontent.py", line 244, in handle_async
    renderable_only=options["renderable_only"])
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/core/content/management/commands/importcontent.py", line 114, in download_content
    self._transfer(DOWNLOAD_METHOD, channel_id, node_ids=node_ids, exclude_node_ids=exclude_node_ids, baseurl=baseurl, renderable_only=renderable_only)
  File "/home/kolibri/.pex/install/kolibri-0.11.0b2-py2.py3-none-any.whl.2ccf04eb4eedaed876a6e0e5508452eeb385b3fe/kolibri-0.11.0b2-py2.py3-none-any.whl/kolibri/core/content/management/commands/importcontent.py", line 180, in _transfer
    raise exception
requests.exceptions.ChunkedEncodingError: ('Connection broken: error("(104, \'ECONNRESET\')",)', error("(104, 'ECONNRESET')",))
```

…

### Steps to reproduce

Not sure what caused this, just random network spotynesss.

### Context

Updating this demo server  sikana-demo.learningequality.org/learn/

**Comment by lyw07** [comment]
Maybe we can add this to the content import retry logic, if it's just caused by the internet?

**Comment by indirectlylit** [comment]
that makes sense to me. Would it make sense to retry for _all_ [requests exceptions](http://docs.python-requests.org/en/master/_modules/requests/exceptions/)? e.g. `RequestException`

**Comment by lyw07** [comment]
@indirectlylit I agree that if we catch `RequestException`, we don't need to worry about keeping adding exceptions into retry logic. But I'm a bit hesitated to do this because `RequestException` contains many exceptions that we haven't seen before, and the number of retries in our logic is unlimited. If, by any chance, an exception that won't be fixed by retrying occurs, users will keep waiting for downloading content until they check the logs. This is just my assumption though. This case may never happen.

Anyway, I think a safe way to fix this specific issue now is to only add `ChunkedEncodingError` to the retry logic. But if we see other exceptions from `RequestException` in the future, we should definitely consider using `RequestException` to replace `ConnectionError`, `Timeout` and `ChunkedEncodingError` at that time.

**Comment by rtibbles** [comment]
Looks like this was fixed in #4541 - please reopen if not the case.
